src/WebJobs.Script.WebHost/Web.config

```diff
@@ -148,7 +148,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.IO.Compression" publicKeyToken="b77a5c561934e089" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.1.2.0" />

```

Comment:
Ok. Just curious, what's going on here? VS makes this update by default when I updated the sdk/extension packages. .